actions/signing-event/action.yml

```diff
@@ -59,60 +59,83 @@ runs:
-          const issues = await github.rest.issues.listForRepo({
-            owner: context.repo.owner,
-            repo: context.repo.repo,
-            labels: [process.env.GITHUB_REF_NAME],
+          const title = `Signing event: ${process.env.GITHUB_REF_NAME}`
+          const repo = `${context.repo.owner}/${context.repo.repo}`
+          const prs = await github.rest.search.issuesAndPullRequests({
+            q: `in:title+"${title}"+state:open+type:pr+repo:${repo}`
           })
-          if (issues.data.length > 1) {
-            core.setFailed("Found more than one issue with same label")
-          } else if (issues.data.length == 0) {
-            const response = await github.rest.issues.create({
+
+          if (prs.data.total_count > 1) {
+            core.setFailed("Found more than one open pull request with same title")
+          } else if (prs.data.total_count == 0) {
+            const response = await github.rest.pulls.create({
               owner: context.repo.owner,
               repo: context.repo.repo,
-              title: "Signing event: " + process.env.GITHUB_REF_NAME,
-              body: "Processing signing event " + process.env.GITHUB_REF_NAME + ", please wait.",
-              labels: [process.env.GITHUB_REF_NAME],
+              title: title,
+              body: `Processing signing event ${process.env.GITHUB_REF_NAME}, please wait.`,
+              draft: true,
+              head: process.env.GITHUB_REF_NAME,
+              base: "main",
             })
-            issue = response.data.number
-            console.log("Created issue #" + issue)
+            pr = response.data.number
+            console.log(`Created pull request #${pr}`)
           } else {
-            issue = issues.data[0].number
-            console.log("Found existing issue #" + issue)
+            pr = prs.data.items[0].number
+            console.log(`Found existing pull request #${pr}`)
           }
 
+          message = fs.readFileSync('./status-output').toString()
           if (process.env.STATUS == 'success') {
-            pr_url = new URL("https://github.com/" + repo + "/compare/main..." + process.env.GITHUB_REF_NAME)
-            pr_url.searchParams.set("expand", "1")
-            pr_url.searchParams.set("title", "Signing event " + process.env.GITHUB_REF_NAME)
-            pr_url.searchParams.set("body", "Signing event " + process.env.GITHUB_REF_NAME + " is successful and ready to merge.\n\nCloses #" + issue + ".")
             message += "### Signing event is successful\n\n"
-            message += "Threshold of signatures has been reached. A [pull request](" + pr_url + ") can be opened."
+            message += "Threshold of signatures has been reached: this signing event can be reviewed and merged."
             summary = "Signing event is successful"
+            should_be_draft = false
           } else {
             summary = "Signing event in progress"
+            should_be_draft = true
```

Comment:
Just my personal preference, ignore if you don't agree. But I would prefer if `should_be_draft = true` is set on line 96 (after `message = fs...`) and then only changed _if_ it should be set to `false`, i.e if event is successful. I just found it a bit strange looking when there is no variable definition in the scope it's later used.